src_new/creative/owCreativeRenderer.js

```diff
@@ -0,0 +1,44 @@
+require("prebid-universal-creative");
+
+window.PWT = window.PWT || {};
+window.pbjs = window.pbjs || {};
+
+var refThis = this;
+
+// Desc: Uses prebid js render ad function to call cache and render ad based on the params from response.
+// Param Defination 
+exports.renderCreative = function (cacheUrl, cachePath, uuid) {
+	if (cacheUrl && uuid) {
+		window.pbjs.renderAd(document, "", {
+			cacheHost: cacheUrl,
+			cachePath: cachePath,
+			uuid: uuid,
+			mediaType: "banner",
+		});
+	}
+};
+
+exports.removeProtocolFromUrl = function(url){
+	var outputUrl = url || "";
+	if(url && url.length > 0){
+		outputUrl = url.replace(/^http:\/\//i, "");
```

Comment:
change to single replacement, 
url.replace(/^https{0,1}:\/\//i, "");

Reply:
Done